Analyzer/plugins/Analyzer.cc

```diff
@@ -4007,6 +4049,21 @@ void Analyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
         tuple->PostPreS_ProbQNoL1->Fill(1 - probQonTrackNoL1, eventWeight_);
         tuple->PostPreS_ProbQNoL1VsIas->Fill(1 - probQonTrackNoL1, globalIas_, eventWeight_);
         tuple->PostPreS_ProbQNoL1VsFiStrips->Fill(1 - probQonTrackNoL1, globalFiStrips_, eventWeight_);
+
+
+        // test --> Need to blind the data at the drawing
+        // for the moment K and C are hard-coded...
```

Comment:
why dont you just take this the usual way btw? the `dEdxK_` and `dEdxC_` are global variables and you would have access to them